ghcide/ghcide.cabal

```diff
@@ -156,6 +155,15 @@ library
         Development.IDE.Core.Tracing
         Development.IDE.Core.UseStale
         Development.IDE.GHC.Compat
+        Development.IDE.GHC.Compat.Core
+        Development.IDE.GHC.Compat.Env
+        Development.IDE.GHC.Compat.Iface
+        Development.IDE.GHC.Compat.Logger
+        Development.IDE.GHC.Compat.Outputable
+        Development.IDE.GHC.Compat.Parser
+        Development.IDE.GHC.Compat.Plugins
+        Development.IDE.GHC.Compat.Units
+        Development.IDE.GHC.Compat.Util
```

Comment:
Probably indeed. But I figured, sometimes it might be nice to have more fine-grained imports.

Reply:
Yeah, I think it's nice to expose them separately, since using `Development.IDE.GHC.Compat` is even more course-grained than using the old GHC api. It's basically re-exporting all the parts of the GHC api that we need compatibility code for (unless that has changed since last I touched it).

Reply:
Exactly. However, it is really non-trivial to have a more fine-grained API.